game/src/test/scala/io/aigar/game/CellSpec.scala

```diff
@@ -224,4 +227,13 @@ class CellSpec extends FlatSpec with Matchers {
 
     opponent.cells should contain only largeCell
   }
+
+  "performAction" should "change target with the right action" in {
```

Comment:
Maybe reword the description a bit here? Idea: `change the target to match the one from the action`

Unrelated: could be interesting to rename `Action` to `Actions`, because it *can* represent multiple actions. Or maybe `PlayerInput`? Not sure.